Web/MVCAssignment/WebContent/View/LandingPage.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
+
+</head>
+
+<body>
+	<form action=" EmployeeDataFetchAndSend" method="get">
+
+		<p>Metacube is a software engineering services company that has
+			deep experience in developing enterprise level products and
+			applications for a wide spectrum of domains including global trade
+			management, supply chain analytics, manufacturing analytics, business
+			continuity planning, CRM, publishing and eCommerce. These
+			applications have been developed on a variety of platforms. The
+			common denominator in all our services has been our total customer
+			focus, ensuring that each engagement is a success and provides the
+			desired value to the customer.</p>
+
+		<a
+			href="http://localhost:8080/MVCAssignment/EmployeeDataFetchAndSend?id=2">Register</a>
+		&nbsp; &nbsp; &nbsp; &nbsp; <a
+			href="http://localhost:8080/MVCAssignment/EmployeeDataFetchAndSend?id=1">List
```

Comment:
ACTION: Don't use static path like http://localhost:8080. because we can use different port to run application and we can run it on remote server.